components/onboarding-modal/OnboardingStepsProgress.js

```diff
@@ -0,0 +1,83 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Flex } from '@rebass/grid';
+
+import { Span } from '../Text';
+import StepsProgress from '../StepsProgress';
+import { Router } from '../../server/pages';
+
+const StepLabel = styled(Span)`
+  text-transform: uppercase;
+  text-align: center;
+`;
+
+StepLabel.defaultProps = {
+  color: 'black.400',
+  fontSize: 'Tiny',
+  mt: 1,
+};
+
+const steps = [{ name: 'Welcome' }, { name: 'Administrators' }, { name: 'Contact' }];
+
+const params = {
+  0: {
+    routerStep: undefined,
+  },
+  1: {
+    routerStep: 'administrators',
+  },
+  2: {
+    routerStep: 'contact',
+  },
+};
+
+class OnboardingStepsProgress extends React.Component {
+  static propTypes = {
+    step: PropTypes.number,
+    slug: PropTypes.string,
+  };
+
+  constructor(props) {
+    super(props);
+    this.setParams = this.setParams.bind(this);
+
+    this.state = {
+      focus: steps[this.props.step],
+    };
```

Comment:
Do we need state here?

Reply:
I have just tested it and apparently not?? Must have been leftover from my first attempts at syncing up the steps and url and stuff.